index.js

```diff
@@ -1,4 +1,4 @@
 const nbind = require('nbind');
-const binding = nbind.init();
+const binding = nbind.init(__dirname);
```

Comment:
Probably this nbind feature is not emphasized enough in the docs... Without the `__dirname` parameter it looks for compiled binaries under `process.cwd()` which will fail if run from outside the root of that specific package, particularly if others are using your bindings in their project.